shopify/product-quiz-template/extensions/quiz/assets/product-quiz.ts

```diff
@@ -1,8 +1,17 @@
-// initialize an API client object
-const api = new Gadget();
+import { Client, GadgetRecordList } from "@gadget-client/product-quiz-template";
```

Comment:
Was this tested at all? Theme extensions do not support `.ts` out of the box
![CleanShot 2024-11-07 at 13 02 47@2x](https://github.com/user-attachments/assets/27b017f8-e261-492c-9e9a-cc2e123d158c)